packages/openapi-to-graphql/test/example_api.test.ts

```diff
@@ -2411,3 +2411,32 @@ test('Non-nullable properties from nested allOf', () => {
     })
   })
 })
+
+test('it formats the request url appropriate when style and explode is set to true', async () => {
+  const LIMIT = 10
+  const OFFSET = 0
+
+  const query = `
+    query {
+      fetchAllOfficesWithFormStyleAndExplodeTrue(parameters: { limit: ${LIMIT}, offset: ${OFFSET} }) {
+        roomNumber
+        company {
+          id
+        }
+      }
+    }
+  `
+
+  await graphql(createdSchema, query).then((result) => {
+    // target error field because the corresponding server url is not implemented,
+    // also we get the full request url as in failed request errors
+    result.errors.forEach((error) => {
```

Comment:
Thank you so much for adding a test case! Adding test cases make my life so much easier.

This is certainly an interesting way to test this functionality, to create an error and examine the request. I think it would be really interesting if we could extract the backend requests that we are making and use them for test cases but I think this pattern isn't the best way to do this.

I am currently working on another issue that I have come across which will also require me to rework this test. We can accept this for now.